app/models/article.rb

```diff
@@ -9,12 +9,14 @@
 #  updated_at :datetime         not null
 #
 class Article < ApplicationRecord
-    has_many :comments,
+    has_many(
+    :comments,
     dependent: :destroy
-    validates :title,
-    presence: true,
-    length: { minimum: 5 }
-    validates :text,
-    presence: true,
-    length: { minimum: 5 }
+    )
+
+    validates(
+    :title,
+    :text,
+    presence: true
```

Comment:
if you are validating the presence of any of your columns,
that means they shouldn't get nil values right?
so, it makes sense to also add NOT NULL constraints for your DB table.

![image](https://user-images.githubusercontent.com/11631898/78820188-0ca50d00-79e0-11ea-85b2-58647b14d91c.png)
see, your db table accepts nil values